partitioned-table.md

```diff
@@ -248,7 +248,7 @@ PARTITION BY RANGE (`id`)
  PARTITION `P_MAXVALUE` VALUES LESS THAN (MAXVALUE))
 ```
 
-Range INTERVAL partitioning also works with [Range COLUMNS](#range-columns-partitioning) partitioning.
+Range INTERVAL partitioning also works with single column [Range COLUMNS](#range-columns-partitioning) partitioning.
```

Comment:
I think this noun phrase is difficult to understand `single column Range COLUMNS partitioning`. Could we state it more clearly? @mjonss